app.json

```diff
@@ -1,33 +1,36 @@
 {
-  "name": "laravel-bootstrap",
-  "scripts": {
-    "postdeploy": "php artisan key:generate && php artisan db:seed"
-  },
-  "env": {
-    "APP_ENV": "production",
-    "APP_DEBUG": "true",
-    "HEROKU_APP_NAME": {
-      "required": true
+    "name": "laravel-bootstrap",
+    "scripts": {
+        "postdeploy": "php artisan key:generate && php artisan db:seed && php artisan app:release"
     },
-    "HEROKU_PARENT_APP_NAME": {
-      "required": true
+    "env": {
```

Comment:
Leave only `APP_DEBUG` and `APP_ENV` as a required field.

Reply:
so HEROKU_PARENT_APP_NAME and other, can i remove it? or you have example?